runs/2018/installfest/info.yml

```diff
@@ -0,0 +1,56 @@
+title: Asteroidy na InstallFestu
+subtitle: ČVUT FEL - installfest.cz
+time: 11:00–13:55
+default_time:
+  start: '11:00'
+  end: '13:55'
+place: ČVUT FEL, Karlovo náměstí 13, budova E, WS room I
+description: Hra v Pythonu pro naprosté začátečníky i pokročilé
+long_description: |
+    Na tomto workshopu pod vedením lektora vytvoříte variantu arkádové hry
+    Asteroids, která poprvé vyšla na konci sedmdesátých let. Využijeme k tomu
+    framework Pyglet, o kterém nejsou potřeba žádné předchozí znalosti.
+
+    Hráč bude ovládat raketku létající vesmírem a střílející na kolem
+    poletující asteroidy. Ale pozor, s asteroidem se samozřejmě můžete i srazit.
+
+    Předpokládá se pouze velmi základní znalost syntaxe Pythonu a základy
+    objektového programování (klidně z jiného jazyka). Základní znalosti
+    z matematiky a fyziky jsou výhodou (setrvačnost, zrychlení, sinus,
+    kosinus...) ale ne nutností, ale nic hardcore nečekejte, předpokládejme
+    vakuum s nulovým třením.
+
+    Vše ostatní se naučíte na workshopu. Pokročilejší Pythonisti mohou 
+    ozvíjet hru různými dalšími směry, takže by se neměli nudit
+    (výbuchy, efekty, životy, štíty...).
+vars:
+  user-gender: m
+  pyladies: false
```

Comment:
`pyladies: false` by měl být default.
Opravdu chceš `user-gender: m`? S defaultním nastavením dostaneš "Možná sis všiml/a „skoku”, když raketa vyletí z okýnka" a "Nepoznáš, kdy jsi prohrál/a nebo vyhrál/a?" – jestli se ti to nelíbí, pobavme se o tom, jak by to mělo být.

Reply:
Přijde mi to hnusné, to je pravda. Ale nevolil jsem to kvůli tomu, jen jsem předpokládal, že opak f je m. Dám obě var tedy pryč.